instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -180,8 +180,26 @@ main()
     cp -rf ${repoDir}/zm-webclient-portal-example/example ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/portals
 
     echo "\t\t***** downloads content *****" >> ${buildLogFile}
-    mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
-    cp -rf ${repoDir}/zm-downloads/. ${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
+    downloadsDir=${repoDir}/zm-build/${currentPackage}/opt/zimbra/${jettyVersion}/webapps/zimbra/downloads
+    mkdir -p ${downloadsDir}
+    cp -rf ${repoDir}/zm-downloads/. ${downloadsDir}
+
+    if [ "${buildType}" == "NETWORK" ]
```

Comment:
Intuitively, it seems wrong to me to have any mention of NETWORK in the FOSS code/scripts.
I'd expect this code to be in zm-network-build.